admin-manual/installation-setup/integrations/handlenet-setup.rst

```diff
@@ -0,0 +1,35 @@
+.. _handlesystem-setup:
+
+======================================
+Using Handle System with Archivematica
+======================================
+
+For more information about Handle System, see the `Handle.Net`_ website.
+
+*On this page*
+
+* :ref:`Connecting Archivematica and Handle.Net <config-handlenet>`
+* :ref:`Binding PIDs to a Handle.Net registry <using-handlenet>`
+
+.. _config-handlenet:
+
+Connecting Archivematica and Handle.Net
+------------------------------------------
+
+Before users begin minting persistent identifiers (PIDs) and binding them to a
+Handle.Net server, you must fill in the Handle server settings in the
+Administration tab.
```

Comment:
I don't think it's accurate to say that the PIDs themselves are bound to a Handle.Net server. Also sounds strange to talk about `users` and then `you` in the next clause. Maybe `Before Archivematica can mint persistent identifiers (PIDs) via requests a Handle.Net server, users must fill in the Handle server settings in the Administration tab.`